File: Solution/Java/AssessmentEntities/SubjectQuestion.java
package com.transflower.tflAssessment.entities;

public class SubjectQuestion {

    private int QuestionId;
    private String Question;
    private int SubjectId;
    private String Subject;

    // Getters and Setters

    public int getQuestionId(){
        return QuestionId;
    }
    
    public void setQuestionId(int QuestionId){
        this.QuestionId = QuestionId;
    }

    public String getQuestion(){
        return Question;
    }
    
    public void setQuestion(String Question){
        this.Question = Question;
    }

    public int getSubjectId(){
        return SubjectId;
    }

    public void setSubjectId(int SubjectId){
        this.SubjectId = SubjectId;
    }

    public String getSubject(){
        return Subject;
    }

    public void setSubject(String Subject){
        this.Subject = Subject;
    }
}

File: Solution/others/CSharp/Sarthak/AssessmentSolution/AssessmentLib/Entities/SubjectQuestion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AssessmentLib.Entities
{
    public class SubjectQuestion
    {
        public int QuestionId { get; set; }
        public string Question { get; set; }
        public int SubjectId { get; set; }
        public string Subject { get; set; }
        
    }
}
